File: Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/elm/executing/IdentifierRefEvaluator.java
package org.opencds.cqf.cql.engine.elm.executing;

import org.opencds.cqf.cql.engine.execution.State;

public class IdentifierRefEvaluator {

    public static Object internalEvaluate(String name, State state) {

        if (name == null) {
            return null;
        }

        return state.resolveIdentifierRef(name);
    }
}

File: Src/java/engine/src/main/kotlin/org/opencds/cqf/cql/engine/elm/executing/IdentifierRefEvaluator.kt
package org.opencds.cqf.cql.engine.elm.executing

import org.opencds.cqf.cql.engine.execution.State

object IdentifierRefEvaluator {
    @JvmStatic
    fun internalEvaluate(name: String?, state: State?): Any? {
        if (name == null) {
            return null
        }

        return state!!.resolveIdentifierRef(name)
    }
}
